src/lib_json/CMakeLists.txt

```diff
@@ -160,7 +164,7 @@ if(BUILD_STATIC_LIBS)
     endif()
 
     set_target_properties(${STATIC_LIB} PROPERTIES
-        OUTPUT_NAME jsoncpp${STATIC_SUFFIX}
+        OUTPUT_NAME jsoncpp
```

Comment:
I would recommend to keep them different (and consistent across versions) to support packaging of dual (shared+static) build.
As you may noticed MSVC generates import lib (`.lib`) for `.dll`, which should not be confused with static `.lib`.